client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -58,11 +58,11 @@ export const NewAccountWaitingPeriodNotice: React.FC = () => (
 	>
 		{ interpolateComponents( {
 			mixedString: __(
-				'Your first payout is held for 7-14 days. {{whyLink}}Why?{{/whyLink}}',
+				'Payout scheduling becomes available after the standard 7-day waiting period for new accounts is complete. {{learnMoreLink}}Learn more{{/learnMoreLink}}',
```

Comment:
Minor: We could move the `7-day` part to a const and use it her and in the settings